docs/content/common-subs/build-core-image.md

```diff
@@ -0,0 +1,9 @@
+<!--build-core-image-start-->
+Build the core image.
+
+```shell
+pwd
+export EXTRA_CORE_TAG=$(date +test%m%d-%H%M%S)
+make kubestellar-image-local
```

Comment:
how long does this take on avg?  Takes forever for me for 3 images locally

Reply:
This only builds for one platform, the native one. Much faster than native + two emulated.

Reply:
awesome - Thank you